### PR TITLE
Fix docker run example for Journalbeat

### PR DESCRIPTION
**Pull request by dedemorton**
Fixes the issue described here:

https://github.com/elastic/beats/pull/10387#issuecomment-462902112

Please respond to my question flagged for REVIEWERS.

### CODE REVIEW

File: libbeat/docs/shared-docker.asciidoc [hunk 1]
```diff
@@ -100,6 +100,10 @@ conventional approach is to provide a configuration file via a volume mount, but
 it's also possible to create a custom image with your
 configuration included.
 
+// REVIEWERS: We don't have a journalbeat version of this file yet. Should I
```
Comment (kvch) [inline libbeat/docs/shared-docker.asciidoc]: Is this file added manually? I see that this file is missing from https://github.com/elastic/beats/pull/9391.

Comment (dedemorton) [reply libbeat/docs/shared-docker.asciidoc]: I think Dan added them manually. @DanRoscigno Do you want to add the config example, or should we do that?

**Comment by DanRoscigno** [comment]
In general I just copied the default file used by Docker images into the new dir I added.  I did this in case there were config changes that we wanted to make to support the getting started experience that we were not comfortable as having for defaults.  I will have a look at it, this is a good time for me to try Journalbeat.

**Comment by DanRoscigno** [comment]
@dedemorton @kvch : Does it make sense to run journalbeat in containers?  I don't know for sure, but running systemd and journald in a container sounds like it breaks the "only run one process per container" goal.

**Comment by kvch** [comment]
Yes, it does make sense. Users are already running Journalbeat in containers to collect the journal entries of the host. The principle sounds good, but in practice, this rather means that use one container for one, distinct purpose. E.g. do not have a container which runs both a database and a web application. Journalbeat serves a single purpose: collecting entries from the host.

**Comment by DanRoscigno** [comment]
Cool, thanks @kvch .  Now I understand.  Is the run command tested?  I would think that the container needs to be run as root to have access to mount that source dir.  You can just copy the file from beats/journalbeat/journalbeat.docker.yml to the deploy/docker dir if it will work.

**Comment by dedemorton** [comment]
@DanRoscigno I did test the run command, but I used sudo. I'll update the docs accordingly.

**Comment by dedemorton** [comment]
@kvch BTW, I thought using the default yml would work, but I did have to specify the input path:

````
journalbeat.inputs:
- paths: ["/var/log/journal"]
````

Is that expected? I can copy the default config to the deploy dir and add it to this PR. Do you want me to set the path as shown above in the default file?

**Comment by kvch** [comment]
That is not expected. Journalbeat connects to the local journal when the list configured in `paths` is empty. And the local journal is either under `/var/log/journal` or `/run/log/journal`. Let me look into it.

**Comment by dedemorton** [comment]
@kvch I've added the example config file. Let me know if you can reproduce the issue I noticed. It's possible I did something wrong.

**Comment by dedemorton** [comment]
@kvch Did you have a chance to test the paths setting? I'm hoping to get this merged soon.

**Comment by dedemorton** [comment]
Input from Noemi:

The correct volumes to attach to the conaitner are: `-v /var/log/journal:/var/log/journal -v /etc/machine-id:/etc/machine-id -v /run/systemd:/run/systemd -v /etc/hostname:/etc/hostname:ro`

**Comment by DanRoscigno** [comment]
I will use those mounts in my kubernetes daemonset also.  Thanks!

**Pull request merged**